sim/Gfluct.mod

```diff
@@ -179,11 +179,6 @@ PROCEDURE oup() {		: use Scop function normrand(mean, std_dev)
 
 VERBATIM
 #include "nrnran123.h"
```

Comment:
We can drop this too, https://github.com/neuronsimulator/nrn/pull/1755 means that it is implicitly included.

Reply:
Done in 399f538654fd91f9152906e95e51b3de5702c28c.